Jenkinsfile

```diff
@@ -127,6 +127,7 @@ node('rhel8'){
 
 		if (publishToMarketPlace.equals('true')) {
 			// Open-VSX Marketplace does not support pre-release
+			sh 'npm install -g "vsce"'
```

Comment:
Why not install this at the beginning of the `node` block (eg. like 106/108)? It looks like `vsce` is run on line line 122, before this install is run

Reply:
ah whoops. this currently does nothing.

Reply:
Done. Should be at the top of the final 'node' declaration that publishes..